regression/ansi-c/CMakeLists.txt

```diff
@@ -1,15 +1,20 @@
 if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
-add_test_pl_tests(
-    "$<TARGET_FILE:goto-cc>" -X gcc-only
-)
+    add_test_pl_tests(
+        "$<TARGET_FILE:goto-cc>" -X gcc-only
+    )
+elseif("${CMAKE_SYSTEM_NAME}" STREQUAL "Darwin")
+    add_test_pl_tests(
+        "$<TARGET_FILE:goto-cc>" -X macos-assert-broken
```

Comment:
Can you double check this? I am not sure the logic is quite correct. As I read this change, the new behaviour on mac will be to not run the c++ front end tests at all, and to skip the C tests, which are marked with `macos-assert-broken`. Or am I reading this incorrectly? I thought we just needed to skip the C++ front end tests on mac which are tagged with `macos-assert-broken` and still run all the C tests on mac.

Reply:
Yep the cmake build looks sorted.